source/content/guides/composer-convert.md

```diff
@@ -25,85 +40,74 @@ export site=yoursitename
 
   ```bash{promptUser:user}
-  git checkout -b composify
+  git remote add ic git@github.com:pantheon-upstreams/drupal-project.git && git fetch ic && git checkout -b composerify ic/master
   ```
 
-  You can replace `composify` with a branch name of your choosing, but all following steps assume this name.
-
-## Set up a Multidev (Optional)
-
-If your Pantheon account has access to [Multidev](/multidev), create a Mmultidev to push your new code to:
-
-```bash{promptUser:user}
-git push origin composify && terminus env:create $site.dev composify
-```
-
-This will set up the Multidev environment to receive and demo our changed code.
-
-## Create a New Composer Project
+  If you prefer, you can replace `composerify` with another branch name. If you do, remember to adjust the other examples in this doc to match.
 
-1. In your local terminal, from the repository root of your Pantheon site, move a directory up:
+1. Copy any existing configuration from the default branch. If no files are copied through this step, that's ok:
 
   ```bash{promptUser:user}
-  cd ..
+  git checkout master sites/default/config
+  git mv sites/default/config/* config
+  git rm -f sites/default/config/.htaccess
+  git commit -m "Pull in configuration from default branch"
   ```
 
-2. Use Composer to create a new project, using the [Pantheon Drupal 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer) repository:
-
-    ```bash{promptUser:user}
-    composer create-project pantheon-systems/example-drops-8-composer $site-composer
-    cd $site-composer
-    ```
+1. Check for `pantheon.yml` settings you need to preserve by comparing your old codebase's `pantheon.yml` to the new `pantheon.upstream.yml`:
 
-This will create a new directory based on the example project [pantheon-systems/example-drops-8-composer](https://github.com/pantheon-systems/example-drops-8-composer) in the `$site-composer` directory.
+  ```bash{promptUser:user}
+  git diff master:pantheon.yml pantheon.upstream.yml
+  ```
 
-### Copy pantheon.upstream.yml
+   - If there are settings from `pantheon.yml` (shown with a `-` in the diff output), consider copying over your old `pantheon.yml` to preserve these settings:
 
-Since the drops-8 upstream has a `pantheon.upstream.yml` and the example-drops-8-composer upstream does not, copy over our old file for the platform to properly load the site. From the `$site-composer` directory, run:
+     ```bash{promptUser:user}
+     git checkout master pantheon.yml
+     git add pantheon.yml
+     git commit -m 'Copy my pantheon.yml'
+     ```
 
-```bash{promptUser:user}
-cp ../$site/pantheon.upstream.yml .
-```
+   If you prefer to keep the values for any of the following settings from `pantheon.upstream.yml`, remove them from `pantheon.yml`:
 
-`ls` should reveal that the new code repository now has a copy of the `pantheon.upstream.yml`.
+     * `web_docroot`
+     * `build_step`
+     * `database`
```

Comment:
So right now the upstream tries to upgrade to 10.4, which isn't going to work for a while. Perhaps we need to instruct folks to set the database version to 10.0 until general availability.

Reply:
I thought our approach to this was to release the documentation for performing conversions to IC only for now, staying on the major version of Drupal you're already on. The title says "Convert to Drupal 9", but we fundamentally can't do that, so we should change the title.

I thought we had an optional step describing what to do to stay on Drupal 8 and we would essentially just make it a required step. But I do not see it in the rendered documentation. Perhaps this is what I had that post-it I had to ping you was about @EdwardAngert -- the details of how to stay on Drupal 8 did change with the new release of the upstream. I'll take a stab at those steps now.

Reply:
The comments are getting numerous and hard to follow, but it looks like I posted a recommended change for what makes sense to me at https://github.com/pantheon-systems/documentation/pull/6145#discussion_r575484057

Reply:
The goal here is to put people on the pantheon-upstreams/drupal-project upstream. We could make an 8.x branch of that upstream, or make a different upstream if we want to simplify the instructions here, but if we keep on the current course, pantheon.upstream.yml sets the database version to 10.4.  It is therefore insufficient for us to remove the database version; we must reset it to 10.0 or the user will receive an error.

Should confirm -- I have not tried this for a while. My understanding, though, is that this error will persist until Mariadb / Drupal 9 general availability.